spyder/widgets/tabs.py

```diff
@@ -48,6 +48,12 @@ def __init__(self, parent, ancestor):
         self.setAcceptDrops(True)
         self.setUsesScrollButtons(True)
 
+        # Tab name editor
+        self.rename_tabs = rename_tabs
+        if self.rename_tabs is True:

```

Comment:
Usually I use this as an implicit "assert isinstance(rename_tabs, bool)" but I'll align with your code style